### PR TITLE
Rarcif issue 55

### PR DESCRIPTION
**Pull request by rarcif**
Issue 55: Please review 

Added:
- RPC Methods
- RPC Server
- RPC Namespaces

### CODE REVIEW

**Comment by lezzokafka** [comment]
Happy first PR @rarcif  🎉🎉

File: docs/resources/chain-integration.md [hunk 1]
```diff
@@ -46,6 +46,266 @@
...
+| `eth_compileSolidity`                                                             | Eth       |             |        |                    
+| `eth_compileSerpent`                                                              | Eth       |             |        |                    
+| `eth_signTransaction`                                                             | Eth       |             |        |                    
+| [`eth_coinbase`](#eth-coinbase)                                                   | Eth       | ✔           |        |                    
+| [`eth_getProof`](#eth-getProof)                                                   | Eth       | ✔           |        |                    
+| [`eth_subscribe`](#eth-subscribe)                                                 | Websocket | ✔           |        |                    
+| [`eth_unsubscribe`](#eth-unsubscribe)                                             | Websocket | ✔           |        |                    
+
+:::tip
+Block Number can be entered as a Hex string, `"earliest"`, ``"latest"`` or `"pending"`.
```
Comment (lezzokafka) [inline docs/resources/chain-integration.md:106]: Can add an extra tip here on the endpoints.
Where the examples below are with a local node `http://localhost:8545`; user can also use our public full node for
`https://cronos-testnet-3.crypto.org:8545`.

File: docs/resources/chain-integration.md [hunk 2]
```diff
@@ -46,6 +46,266 @@
...
+```json
+// Request
+curl -X POST --data '{"jsonrpc":"2.0","method":"eth_gasPrice","params":[],"id":1}' -H "Content-Type: application/json" http://localhost:8545
+// Result
+{"jsonrpc":"2.0","id":1,"result":"0x0"}
+```
+
+### `eth_accounts`
+
+Returns array of all eth accounts.
```
Comment (lezzokafka) [inline docs/resources/chain-integration.md:211]:
```suggestion
Returns array of all eth accounts owned by the client.
```

File: docs/resources/chain-integration.md (hunk 1 above)
Comment (rarcif) [reply docs/resources/chain-integration.md]: Added

File: docs/resources/chain-integration.md (hunk 2 above)
Comment (rarcif) [reply docs/resources/chain-integration.md]: updated

File: docs/resources/chain-integration.md [hunk 3]
```diff
@@ -46,6 +46,268 @@
...
+
+### JSON-RPC Support
+
+Ethermint supports all standard web3 JSON-RPC APIs. You can find documentation for these APIs on the [`JSON-RPC Methods`](./endpoints.md) page.
+
+JSON-RPC is provided on multiple transports. Ethermint supports JSON-RPC over HTTP and WebSocket. Transports must be enabled through command-line flags or through the `app.toml` configuration file. For more details see the []
+
+Ethereum JSON-RPC APIs use a name-space system. RPC methods are grouped into several categories depending on their purpose. All method names are composed of the namespace, an underscore, and the actual method name within the namespace. For example, the eth_call method resides in the eth namespace.
+
+Access to RPC methods can be enabled on a per-namespace basis. Find documentation for individual namespaces in the [Namespaces](./namespaces.md) page.
```
Comment (lezzokafka) [inline docs/resources/chain-integration.md:242]: [Namespaces](./namespaces.md) should be linked to `## JSON-RPC namespaces`?

File: docs/resources/chain-integration.md [hunk 4]
```diff
@@ -46,6 +46,268 @@
...
+
+[JSON](https://json.org/) is a lightweight data-interchange format. It can represent numbers, strings, ordered sequences of values, and collections of name/value pairs.
+
+[JSON-RPC](http://www.jsonrpc.org/specification) is a stateless, light-weight remote procedure call (RPC) protocol. Primarily this specification defines several data structures and the rules around their processing. It is transport agnostic in that the concepts can be used within the same process, over sockets, over HTTP, or in many various message passing environments. It uses JSON ([RFC 4627](https://www.ietf.org/rfc/rfc4627.txt)) as data format.
+
+### JSON-RPC Support
+
+Ethermint supports all standard web3 JSON-RPC APIs. You can find documentation for these APIs on the [`JSON-RPC Methods`](./endpoints.md) page.
+
+JSON-RPC is provided on multiple transports. Ethermint supports JSON-RPC over HTTP and WebSocket. Transports must be enabled through command-line flags or through the `app.toml` configuration file. For more details see the []
```
Comment (lezzokafka) [inline docs/resources/chain-integration.md:238]: Expecting a link to some details pages?

**Review by lezzokafka — COMMENTED** [review]
Left some minor comments, looks great in general 👍

File: docs/resources/chain-integration.md (hunk 3 above)
Comment (rarcif) [reply docs/resources/chain-integration.md]: added

File: docs/resources/chain-integration.md (hunk 4 above)
Comment (rarcif) [reply docs/resources/chain-integration.md:241]: added link and changed wording to section since the endpoints are on the same page but different section

**Review by lezzokafka — APPROVED** [review]

**Pull request merged**